cats-and-dogs/src/components/Dogs/RandomImg.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import axios from "axios";
+import DisplayDog from "./DisplayDog"
+
+class Random extends React.Component{
+   
+    state = {img:[],
+            num:0
+    }
+
+    handleNum = (e) => {
+        this.setState({
+            num: e.target.value
+        })
+    }
+
+    handleSubmit = (e) => {
+        e.preventDefault()
+        this.getMultiple()
+
+    }
+
+   getMultiple = async (e) => {
+      let { num } = this.state
+
+       try{
+        let res = await axios.get(`https://cors-anywhere.herokuapp.com/https://dog.ceo/api/breeds/image/random/${num}`)
+        this.setState({img:res.data.message})
+       }catch(err){
+           console.log(err)
```

Comment:
In the case of an error setting the img state to be empty would be a good way to deal with it.